tests/l2/conftest.py

```diff
@@ -0,0 +1,11 @@
+# conftest.py for L2 configuration.
+
+
+def pytest_addoption(parser):
```

Comment:
Nightly test is used to verify specified sonic version, if we upgrade sonic image in this test, we can't add test to nightly test.

Reply:
This is what the issue is asking so if we can't upgrade, we need to find another way to trigger db_migrator?

Btw the test will restore the original image. Is that good enough?

Reply:
Please confirm with Vaibhav, I guess we need to upgrade from 20191130.xx to 20240531.xx and verify L2 configuration.
If we do this in nightly test for 20240531.xx, how do you choose new image to upgrade?

Reply:
I'll check with him. Maybe we should do a --source_image and --target_image. It will make more sense to upgrade from an older image to 20240531 in the nightly test.

Reply:
Checked with Vaibhab, this test should be in nightly. I will rewrite part of the test according to [test_upgrade_path.py](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/upgrade_path/test_upgrade_path.py) to better accommodate that.